.github/workflows/merge-dependabot.yml

```diff
@@ -0,0 +1,15 @@
+
+name: auto-merge
+on:
+  pull_request:
+    types: [ labeled ]
+jobs:
+  auto-merge:
+    if: ${{ github.event.label.name == 'dependencies' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
```

Comment:
why `minor` here? I am expecting `patch`

Reply:
Updated to `patch`.